SayTheirNames/Source/Controller/Petitions/PetitionTableViewCell.swift

```diff
@@ -0,0 +1,218 @@
+//
+//  PetitionTableViewCell.swift
+//  Say Their Names
+//
+//  Created by Joseph A. Wardell on 6/4/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+class PetitionTableViewCell: UITableViewCell {
+
+    var findOutMoreAction : () -> Void = {}
+
+    lazy var container: UIView = {
```

Comment:
Just a few areas in here that i'd recommend using `private`

Reply:
agreed. done. thanks